collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -39,14 +40,26 @@
         'lines': [
             ['users', 'Users', 'absolute', 1, 1]
         ]
+    },
+    'mem': {
+        'options': [None, 'Minecraft Memory Usage', 'MiB', 'spigotmc', 'spigotmc.mem', 'line'],
+        'lines': [
+            ['mem_cur', 'current', 'absolute', 1, 1],
+            ['mem_total', 'total', 'absolute', 1, 1],
+            ['mem_max', 'max', 'absolute', 1, 1]
```

Comment:
There is a good article about [runtime memory functions](https://newbedev.com/what-are-runtime-getruntime-totalmemory-and-freememory). Not a specialist, but i think it would be better to do the following changes:
 - current => used
 - total => allocated

What do you think?